### DISCUSSION TITLE
Callback for the slider in a plugin

### DISCUSSION THREAD

**Discussion by Clay-CL** [facontidavide/PlotJuggler#975] [Q&A]
Is there a callback available in plotjuggler plugins to obtain a "tick" everytime the slider is moved?
This would be useful when running plotjuggler alongside another app, the tiimestamp data could be communicated via IPC to that program, which in turn can use it to index the same data but might perform some other processing of it.

Would be great to hear everyone's thoughts on this
Thanks

**Comment by sinnanodelta** [comment]
Hi :)

A very simple way is to send the current timestamp (by UDP, via QUdpSocket, for example) from a custom plugin (based on the StatePublisher class). It allows to a third-party software to read it and trigger whatever you need.

Hope it helps
